pkg/aws/admin-session.go

```diff
@@ -0,0 +1,187 @@
+	fmt.Print("Logging out of AWS to refresh privileges\n\n")
+	err = doAWSLogout()
+	if err != nil {
+		return err
+	}
+
+	printDivider()
+
+	fmt.Print("\nStart SSO Login\n\n")
+	err = doAWSLogin(awsProfile)
+	if err != nil {
+		return err
+	}
+	fmt.Println()
+
+	printDivider()
+
+	fmt.Print("\nVerifying selected AWS profile by listing S3 buckets\n\n")
+	err = listS3Buckets(awsProfile)
+	fmt.Println()
+	printDivider()
+	if err != nil {
+		fmt.Print("\n", red.Render("Blaah!! You don't have the correct rights!"), "\n\n")
+		return cleanupAndQuit()
+	}
+
+	fmt.Print("\n", green.Render("Great! Access granted"), "\n\n")
+	fmt.Print("Remove your Access Package when done (or extend if needed):\n")
+	fmt.Print(yellow.Render("https://myaccess.microsoft.com/@oslokommune.onmicrosoft.com#/access-packages/active"), "\n\n")
+
+	fmt.Print("After the Access Package is disabled, please log out of current session.\n\n")
+	fmt.Print("Easily done with: ", yellow.Render("aws sso logout"), "\n\n")
+	fmt.Print("Take care - have fun!\n")
+	return nil
+}
+
+func selectAWSProfile() (string, error) {
+	cmd := exec.Command("aws", "configure", "list-profiles")
+	cmd.Stderr = os.Stderr
+	out, err := cmd.Output()
+	if err != nil {
+		return "", err
+	}
+	var profiles []huh.Option[string]
+	for _, profile := range strings.Split(string(out), "\n") {
+		profiles = append(profiles, huh.NewOption(profile, profile))
+	}
+
+	var selectedProfile string
+	selector := huh.NewSelect[string]().
+		Title("Select AWS profile:").
+		Options(profiles...).
+		Validate(func(t string) error {
+			if len(t) <= 0 {
+				return fmt.Errorf("you need to select a profile")
+			}
+			return nil
+		}).
+		Value(&selectedProfile)
+
+	if err := selector.Run(); err != nil {
+		return "", err
+	}
+	return selectedProfile, nil
+}
+
+func doAWSLogout() error {
+	cmd := exec.Command("aws", "sso", "logout")
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	err := cmd.Run()
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func doAWSLogin(awsProfile string) error {
+	cmd := exec.Command("aws", "sso", "login")
+	cmd.Env = append(os.Environ(), "AWS_PROFILE="+awsProfile)
+	cmd.Stderr = os.Stderr
+	out, err := cmd.StdoutPipe()
+	if err != nil {
+		return err
+	}
+	go handleAWSLoginOutput(out)
+	if err := cmd.Start(); err != nil {
+		return err
+	}
+	if err := cmd.Wait(); err != nil {
+		return err
+	}
+	return nil
+}
+
+func handleAWSLoginOutput(reader io.Reader) {
+	br := bufio.NewReader(reader)
+	for {
+		line, _, err := br.ReadLine()
+		if err != nil {
+			if err == io.EOF {
+				return
+			}
+			panic(err)
+		}
+		fmt.Printf("%s\n", line)
+		if len(line) == 9 {
+			err := sendMacNotification("Admin Session", "Login Code: "+string(line))
+			if err != nil {
+				panic(err)
+			}
+		}
+	}
+}
+
+func listS3Buckets(awsProfile string) error {
+	cmd := exec.Command("aws", "s3", "ls")
+	cmd.Env = append(os.Environ(), "AWS_PROFILE="+awsProfile)
+	cmd.Stdout = os.Stdout
```

Comment:
Siden vi ikke er interessert i å faktisk liste bøttene, men å sjekke at du har tilgang til AWS-kontoen, anbefaler jeg at vi ikke printer ut stdout.
```suggestion
	cmd.Stdout = io.Discard
```